.circleci/generate.sh

```diff
@@ -116,49 +116,53 @@ if $has_env_vars && $check_env_vars; then
   done
 fi
 
+lowers() {
+  circleci config process $BASEDIR/config-2_1.yml > $BASEDIR/config.yml.LOWRES.tmp
+  cat $BASEDIR/license.yml $BASEDIR/config.yml.LOWRES.tmp > $BASEDIR/config.yml.LOWRES
+  rm $BASEDIR/config.yml.LOWRES.tmp
+}
+
+midres() {
+  $BASEDIR/circle-ci-config.py MIDRES --stdout > $BASEDIR/config-2_1.yml.MIDRES
+  circleci config process $BASEDIR/config-2_1.yml.MIDRES > $BASEDIR/config.yml.MIDRES.tmp
+  cat $BASEDIR/license.yml $BASEDIR/config.yml.MIDRES.tmp > $BASEDIR/config.yml.MIDRES
+  rm $BASEDIR/config-2_1.yml.MIDRES $BASEDIR/config.yml.MIDRES.tmp
+}
+
+highers() {
+  $BASEDIR/circle-ci-config.py HIGHER --stdout > $BASEDIR/config-2_1.yml.HIGHRES
+  circleci config process $BASEDIR/config-2_1.yml.HIGHRES > $BASEDIR/config.yml.HIGHRES.tmp
+  cat $BASEDIR/license.yml $BASEDIR/config.yml.HIGHRES.tmp > $BASEDIR/config.yml
+  rm $BASEDIR/config-2_1.yml.HIGHRES $BASEDIR/config.yml.HIGHRES.tmp
+}
```

Comment:
there were slight differences between the single case and the `-a` case, so by leveraging functions we can do this in a consistent way